p4-16/spec/P4-16-spec.mdk

```diff
@@ -5021,6 +5021,8 @@ is captured by the following pseudo-code:
 ~ Begin P4Pseudo
 void packet_in.extract<T>(out T headerLvalue,
                           in bit<32> variableFieldSize) {
+   // implementations _may_ include the following line, but need not
+   verify(variableFieldSize[2:0] == 0, error.ParserInvalidAlignment);
```

Comment:
I think I would prefer this check to be commented-out.

Reply:
done